docs/source/whatsnew/1.2.0.txt

```diff
@@ -0,0 +1,115 @@
+Release 1.2.0
+-------------
+
+:Release: 1.2.0
+:Date: TBD
+
+Extensible Risk and Performance Metrics (:issue:`2081`)
+```````````````````````````````````````````````````````
+
+The risk and performance metrics are summarizing values calculated by Zipline
+when running a simulation, for example: returns or Sharpe ratio. 1.1.2
+introduces a new API for registering custom risk and performance metrics defined
+by the user. We have also made it possible to run a backtest without computing
+any metrics to improve the feedback cycle when debugging an algorithm.
+
+For more information, see :ref:`metrics`.
```

Comment:
And rebased against master for the above ^^

Reply:
Nice, what does this link to?

Reply:
That links to the `Risk and Performance Metrics` section of the docs that Joe added in his risk metrics PR a while back.

I tested the link locally and it works 🙂

Reply:
Oh, awesome!